endpoints/citation.py

```diff
@@ -0,0 +1,35 @@
+from io import BytesIO
+
+from PIL import Image, ImageDraw
+from flask import send_file
+
+from utils.endpoint import Endpoint, setup
+from utils.textutils import wrap
+
+
+@setup
+class Citation(Endpoint):
+    params = ['text']
+
+    def generate(self, avatars, text, usernames, kwargs):
+        try:
+            text = text.replace(', ', ',').split(',')
+        except IndexError:

```

Comment:
This doesn't look like it throws IndexError

Reply:
Yup, def won't throw an error.